app/src/main/java/chat/rocket/android/activity/MainPresenter.java

```
@@ -159,6 +161,19 @@ private void subscribeToSession() {
         );
 
     addSubscription(subscription);
+
+    addSubscription(
+            RxJavaInterop.toV2Observable(connectivityManagerApi.getServerConnectivityAsObservable())
+                    .subscribeOn(Schedulers.io())
+                    .observeOn(AndroidSchedulers.mainThread())
+                    .subscribe(serverConnectivity -> {
+                      if (serverConnectivity.state == ServerConnectivity.STATE_CONNECTING) {
+                        view.showConnecting();
+                      }
+                    },
+                    err -> {
```

Don't ignore exceptions, do at least a `err.printStackTrace();` or if it is really safe to ignore it, add a comment explaining why.

Ignoring exceptions can make it harder to understand why something is not working, and printing the stacktrace make it obvious where the problem is,

---

I agree. I don't have intention to keep it this way. I agree it's bad and the actual codebase is ignoring exceptions wide spread. Just kept this way for the sake of rushing to release this PR for testing. The code needs much refining